Automattic/wp-calypso
# DevDocs: Move Checklist from Components to Blocks

See #24104 

Testing: 

* Switch to this PR
* Navigate to http://calypso.localhost:3000/devdocs/
* Confirm that the `Checklist` has moved from `Components` to `Blocks`
